src/main/resources/static/css/dashboard.css

```
@@ -937,6 +945,20 @@ body {
 }
 
+.sensor-upload-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 8px;
+    padding: 1rem 0;
+}
+
+.sensor-upload-label {
+    font-family: "Gaegu", inherit;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Gaegu" (font-family-name-quotes)

(font-family-name-quotes)